README.md

```diff
@@ -64,4 +64,44 @@ matching the pattern ``test_*`` or ``*_test``, in any file matching the same pat
 
 To run the selenium, please ensure to add the selenium chromedriver on your $PATH. Then run, ``poetry run pytest tests_e2e``. Remember, that you must have Chrome web browser installed.
 
-Note that the selenium tests creates a ``DummyBoard`` on Trello which gets deleted after test completion.
\ No newline at end of file
+Note that the selenium tests creates a ``DummyBoard`` on Trello which gets deleted after test completion.
+
+## Running on Docker
+
+There are two environments that can be run on docker: development and production. The simplest way to launch the app is by running:
+```bash
+# To run both development and production
+docker-compose up 
```

Comment:
Nice! It would be great to also get this to handle the bind-mount that you've added in the instruction for running it manually, but this made it really easy for me to run your containers

Reply:
Thanks, done.